x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit/threshold_watch_edit.tsx

```diff
@@ -716,7 +717,7 @@ const ThresholdWatchEditUi = ({ intl, pageTitle }: { intl: InjectedIntl; pageTit
                                   errors={errors}
                                 >
                                   <EuiFieldNumber
-                                    value={watch.threshold[i]}
+                                    value={watch.threshold[i] || ''}
```

Comment:
If `watch.threshold[i]` is 0 then this will set `value` to the empty string. Is that what we want? Or should we check for `undefined` and `null` instead of just falsiness, e.g. `watch.threshold[i] == null`.

Reply:
good catch, fixed.